app/controllers/concerns/foreman/controller/authentication.rb

```diff
@@ -7,11 +7,11 @@ def available_sso
 
   def authenticate
     return true if (User.current && Rails.env.test? && api_request?) ||
-                   session[:user] && (User.current = User.unscoped.find_by(id: session[:user]))
+                   session[:user] && (User.current = User.unscoped.find_by(id: session[:user])) && !User.current.disabled
 
```

Comment:
Shouldn't we return here is we already have `User.current`, not to continue to the `sso_authentication` and actually inform the user about he's disabled.

Reply:
@domitea could you answer?